chart/localstack/templates/service.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "localstack.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "localstack.name" . }}
+    helm.sh/chart: {{ include "localstack.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - name: http
+      protocol: TCP
+      port: {{ .Values.service.port }}
+      targetPort: http
+      {{- if contains "NodePort" .Values.service.type }}
+      {{- if .nodePort }}
+      nodePort: {{ .nodePort }}
+      {{- end }}
+      {{- end }}
+  {{- range $index, $val := .Values.localstack.services }}
+  {{- if $val.enabled }}
+    - name: {{ $val.name }}
+      protocol: TCP
+      port: {{ $val.externalPort }}
+      targetPort: {{ $val.internalPort }}
+      {{- if contains "NodePort" .Values.service.type }}
```

Comment:
This creates a scope error:
`Error: template: localstack/templates/service.yaml:28:40: executing "localstack/templates/service.yaml" at <.Values.service.type>: nil pointer evaluating interface {}.service`


I think it should be 
`$.Values.service.type`